miscDeps

```diff
@@ -1 +1 @@
-Subproject commit 239621041225ce15b93d6aeb96945eaf67e10898
+Subproject commit a7f918c421e91f1b1601ab822ede2c869275ef5c
```

Comment:
You probably did git commit -a, added all files with git add or 
something like that, but you had an outdated miscDeps submodule at the 
time. Careful observance of git status and/or git show after the commit 
will prevent such things.